e2e/capacity/replenishment.test.ts

```diff
@@ -37,11 +36,11 @@ describe('Capacity Replenishment Testing: ', function () {
     // Replenishment requires the epoch length to be shorter than testnet (set in globalHooks)
     if (isTestnet()) this.skip();
 
-    schemaId = await getOrCreateGraphChangeSchema(fundingSource);
   });
 
   describe('Capacity is replenished', function () {
     it('after new epoch', async function () {
+      const schemaId = await getOrCreateGraphChangeSchema(fundingSource);
```

Comment:
`getOrCreateGraphChangeSchema`, when in the before() and run against a local chain, would fail with either nonce too low or priority errors.  Moving the cal inside each test fixed it.